protocol/hello/hello.go

```diff
@@ -33,13 +33,17 @@ var log = logging.Logger("/fil/hello")
 
 // Message is the data structure of a single message in the hello protocol.
 type Message struct {
-	HeaviestTipSetCids   []cid.Cid
+	HeaviestTipSetCids   types.TipSetKey
```

Comment:
:+1: this brings us inline with the [hello protocol definition](https://github.com/filecoin-project/specs/blob/94a3cbbc6c1b3f848957730f5fff860ec8db1907/network-protocols.md#hello-handshake), nice!